components/library/libjpeg-turbo/libjpeg-turbo.p5m

```diff
@@ -29,18 +29,27 @@ depend fmri=image/library/libjpeg8-turbo type=require
 file path=usr/share/doc/libjpeg-turbo/LICENSE.md
 file path=usr/share/doc/libjpeg-turbo/README.ijg
 file path=usr/share/doc/libjpeg-turbo/README.md
-file path=usr/share/doc/libjpeg-turbo/example.c
+file path=usr/share/doc/libjpeg-turbo/example.txt
 file path=usr/share/doc/libjpeg-turbo/libjpeg.txt
 file path=usr/share/doc/libjpeg-turbo/structure.txt
+file path=usr/share/doc/libjpeg-turbo/tjexample.c
 file path=usr/share/doc/libjpeg-turbo/usage.txt
 file path=usr/share/doc/libjpeg-turbo/wizard.txt
 
-# Already shipped by libjpeg6-ijg
-#file path=usr/share/man/man1/cjpeg.1
-#file path=usr/share/man/man1/djpeg.1
-#file path=usr/share/man/man1/jpegtran.1
-#file path=usr/share/man/man1/rdjpgcom.1
-#file path=usr/share/man/man1/wrjpgcom.1
+# Previously shipped by libjpeg6-ijg
+file path=usr/share/man/man1/cjpeg.1
+file path=usr/share/man/man1/djpeg.1
+file path=usr/share/man/man1/jpegtran.1
+file path=usr/share/man/man1/rdjpgcom.1
+file path=usr/share/man/man1/wrjpgcom.1
+
+# Add symlinks to default binaries
+link path=usr/bin/cjpeg target=../lib/libjpeg8-turbo/bin/$(MACH64)/cjpeg
```

Comment:
If you want to make it in a proper way, add optional dependency on libjpeg6-jig version with removed files.

Reply:
Thank you :) I'll do that.